File: csharp/Airports/Airport.cs
using System.Text.Json.Serialization;

namespace ParagonCodingExercise.Airports
{
    public class Airport
    {
        [JsonPropertyName("identifier")]
        public string Identifier { get; set; }

        [JsonPropertyName("longitude")]
        public double Longitude { get; set; }

        [JsonPropertyName("latitude")]
        public double Latitude { get; set; }

        [JsonPropertyName("elevation")]
        public int Elevation { get; set; }
    }
}

File: java/src/com/paragonintel/codingexercise/Airports/Airport.java
package com.paragonintel.codingexercise.Airports;

public class Airport {
    private String identifier;
    private double longitude;
    private double latitude;
    private int elevation;

    public String getIdentifier() {
        return this.identifier;
    }

    public void setIdentifier(String identifier) {
        this.identifier = identifier;
    }

    public double getLongitude() {
        return this.longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }

    public double getLatitude() {
        return this.latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public int getElevation() {
        return this.elevation;
    }

    public void setElevation(int elevation) {
        this.elevation = elevation;
    }
}
